softhsm/Dockerfile.tester

```diff
@@ -0,0 +1,3 @@
+from alpine
```

Comment:
Is this dockerfile used?

Reply:
Yeah, by the docker-compose if you'd like to test.